docs/pc-intel-x86_32/supported-os.rst

```diff
@@ -13,5 +13,10 @@ long as it:
 The following guest OSes are known to work:
 
 
-* Ubuntu 16.04 LTS with Linux Kernel 4.4.x (CONFIG_X86_PAE=n) and below
+* Ubuntu 16.04 LTS with Linux Kernel 4.4.x (#CONFIG_X86_PAE is not set) and below
 * Ubuntu 12.04 LTS with Linux Kernel 3.2.0-27-generic and below
+
+Note, that currently certain kernel modules must be black-listed (e.g., in `//etc/modprobe.d/blacklist.conf`):
```

Comment:
there is an extra `/` within `//etc...`?

Reply:
thanks, removed